cases/filters.py

```diff
@@ -48,26 +48,11 @@ def __init__(self, data, *args, **kwargs):
         data = data.copy()
         user = kwargs["request"].user
         submitted = data.get("kind") is not None
-        # If the user has associated wards, and we've been asked for all/unassigned/others
-        # (but not via a filter form submission), use those wards by default
-        if (
-            user.wards
-            and not submitted
-            and data.get("assigned") in ("", "none", "others")
-        ):
-            data.setlistdefault("ward", user.wards)
-        # Default to showing cases assigned to the user
-        data.setdefault("assigned", "me")
         super().__init__(data, *args, **kwargs)
         self.filters.move_to_end("search", last=False)
         self.filters["kind"].label = "Noise type"
         self.filters["where"].label = "Noise location type"
         self.filters["estate"].label = "Hackney Estates property?"
-        try:
```

Comment:
Not sure why you've got rid of this part - this is so links on people's names to see their assigned stuff (so assigned=123) work.

Reply:
When I left it in, I was getting a `MultiValueDictKeyError` exception on loading `/cases`:

```web_1      | Internal Server Error: /cases
web_1      | Traceback (most recent call last):
web_1      |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/utils/datastructures.py", line 76, in __getitem__
web_1      |     list_ = super().__getitem__(key)
web_1      | KeyError: 'assigned'
web_1      | 
web_1      | During handling of the above exception, another exception occurred:
web_1      | 
web_1      | Traceback (most recent call last):
web_1      |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
web_1      |     response = get_response(request)
web_1      |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
web_1      |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
web_1      |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
web_1      |     return view_func(request, *args, **kwargs)
web_1      |   File "/app/cases/views.py", line 42, in case_list
web_1      |     return case_list_staff(request)
web_1      |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
web_1      |     return view_func(request, *args, **kwargs)
web_1      |   File "/app/cases/views.py", line 65, in case_list_staff
web_1      |     f = CaseFilter(request.GET, queryset=qs, request=request)
web_1      |   File "/app/cases/filters.py", line 57, in __init__
web_1      |     user = User.objects.get(id=data["assigned"])
web_1      |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/utils/datastructures.py", line 78, in __getitem__
web_1      |     raise MultiValueDictKeyError(key)
web_1      | django.utils.datastructures.MultiValueDictKeyError: 'assigned'
web_1      | [11/Jan/2022 08:51:12] "GET /cases HTTP/1.1" 500 97842```

Reply:
Ah, you want to keep it (for its functionality), but because of what you've removed above it (which made sure assigned was always set), you need to change `user = User.objects.get(id=data["assigned"])` to `user = User.objects.get(id=data.get("assigned", ""))` - I think. If that doesn't work I can take a look.

Reply:
Thanks! Have pushed a fixup commit.